README.md

```diff
@@ -1,15 +1,15 @@
 PHPainfree
 ==========
 
-PHPainfree is an **ultra**-lightweight PHP framework. It is inspired by the
-MVC concept and attempts to remove barriers to entry while providing the
+PHPainfree was an **ultra**-lightweight PHP framework. It was inspired by the
+MVC concept and attempted to remove barriers to entry while providing the
```

Comment:
We need to add more adjectives describing how GREAT PHPainfree is, this feels a little 📉  and we need more 📈.